### PR TITLE
[Feature](serialize) move block_data_version to fe heart beat

### PR DESCRIPTION
**Pull request by BiteTheDDDDt**
# Proposed changes

Move block_data_version from be config to fe heart beat

## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [ ] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [ ] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [ ] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by HappenLee — DISMISSED** [review]
LGTM

**Review by morningman — CHANGES_REQUESTED** [review]
Please add some document for this config

**Comment by BiteTheDDDDt** [comment]
> Please add some document for this config

done

File: fe/fe-core/src/main/java/org/apache/doris/common/Config.java [hunk 1]
```diff
@@ -1767,4 +1767,22 @@ public class Config extends ConfigBase {
...
+    /**
+     * Min data version of backends serialize block.
+     */
+    @ConfField(mutable = false)
+    public static int min_block_data_version = 0;
+
+    /**
+     * Data version of backends serialize block.
+     */
+    @ConfField(mutable = true)
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/common/Config.java:1786]:
```suggestion
    @ConfField(mutable = true, masterOnly = true)
```

**Review by HappenLee — APPROVED** [review]
LGTM

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**